tools/fixturegen/README.md

```diff
@@ -0,0 +1,25 @@
+# Fixture Generator
```

Comment:
@kwridan would you mind moving this to the documentation website? We can start a new page about performance and there you can explain how to use this utility to check the performance of Tuist generating projects.